ui/analyse/css/_layout.scss

```diff
@@ -77,7 +77,7 @@ body {
     margin: $analyse-block-gap 0 0 0;
   }
 
-  grid-template-rows: auto auto auto minmax(20em, 30vh);
+  grid-template-rows: auto auto auto minmax(20em, 36vh);
```

Comment:
<img width="513" alt="Screenshot 2025-02-27 at 3 16 53 PM" src="https://github.com/user-attachments/assets/b3006142-0ead-45b9-a2b3-570344f051bc" />

In the firefox dev tools, I picked iPhone SE 2nd gen in device selector and saw this. Choosing a maximum height for this grid area that scales based on viewport height when the div is not scrollable might not be best, but adding another nested scrolling area clearly isn't preferable. So not sure how to fix besides bumping up this number or changing its units. Looks like `40vh` works all right.